### PR TITLE
Matches the design team output

### PR DESCRIPTION
**Pull request by CarlosNihelton**
This PR updates the slide show contents to match the output from the design team. 

The window has been scaled a little bit and forced centering the user's main display as a partial workaround to ensure the OOBE will not catch user's attention before the right time. There is another side of this work on WSL launcher itself to find and control the visibility of the OOBE window as the main way of reaching this aim, so the window resizing and positioning is just a secondary measure.

Design screenshots and text can be found in the document:
https://docs.google.com/document/d/16JdUzXox6bYDrsrrDvZ66LJskhy6qBxVyWBh2J7RobM/preview

This is how it looks now:

https://user-images.githubusercontent.com/11138291/161046142-b61c505d-6e65-449c-bfb1-822ffc8e6cdf.mp4

### CODE REVIEW

File: windows/runner/win32_window.cpp [hunk 1]
```diff
@@ -128,10 +128,13 @@ bool Win32Window::CreateAndShow(const std::wstring& title,
     return false;
   }
 
-  const int ubuntuDefaultWidth = 960;
-  const int ubuntuDefaultHeight = 680;
-  ::SetWindowPos(window, nullptr, Scale(origin.x, scale_factor), Scale(origin.y, scale_factor),
-                   ubuntuDefaultWidth, ubuntuDefaultHeight, SWP_NOACTIVATE | SWP_NOOWNERZORDER);
+  const int ubuntuDefaultWidth = static_cast<int>(960*1.025);
+  const int ubuntuDefaultHeight =  static_cast<int>(680*1.05);
```
Comment (jpnurmi) [inline windows/runner/win32_window.cpp:132]: any comment where/how/why 1.025 and 1.05 came from? :)

File: windows/runner/win32_window.cpp [hunk 2]
```diff
@@ -128,10 +128,13 @@ bool Win32Window::CreateAndShow(const std::wstring& title,
...
+  int x = (GetSystemMetrics(SM_CXSCREEN)-ubuntuDefaultWidth)/2;
```
Comment (jpnurmi) [inline windows/runner/win32_window.cpp:133]: nitpick: spaces around `-` and `/`

File: windows/runner/win32_window.cpp (hunk 1 above)
Comment (jpnurmi) [reply windows/runner/win32_window.cpp:132]: nitpick: spaces around `*`

File: lib/slide.dart [hunk 3]
```diff
@@ -73,25 +105,80 @@ class Slide extends StatelessWidget {
...
+  final bodyStyle = _bodyTextStyle(context);
+  final linkStyle = bodyStyle.copyWith(
+    color: Colors.white,
+    decoration: TextDecoration.underline,
+  );
   return [
     Slide(
-      image: theSameAsset,
+      image: SvgPicture.asset('assets/1-Ubuntu on WSL.svg'),
+      title: lang.welcome,
```
Comment (jpnurmi) [inline lib/slide.dart:118]: lang.welcome needs updating? it's still "Welcome to Ubuntu"

**Review by jpnurmi — COMMENTED** [review]
Nice! I noticed a couple of visual issues.

- The window opens partly outside the screen on my Windows machine
image

- There's an overflow if the output is expanded when the multiline error appears
image

File: lib/slide.dart (hunk 3 above)
Comment (CarlosNihelton) [reply lib/slide.dart:118]: Good catch! It was supposed to be "Welcome to Ubuntu WSL". Thanks. I'll get those issues fixed.

**Comment by jpnurmi** [comment]
There's still something off with the positioning:

image

Here are the relevant values (it's a Surface Laptop 4):
- mInfo.right: 2256
- mInfo.bottom: 1432
- scale_factor: 1.5

**Comment by CarlosNihelton** [comment]
> There's still something off with the positioning:
> 
> image
> 
> Here are the relevant values (it's a Surface Laptop 4):
> 
> * mInfo.right: 2256
> * mInfo.bottom: 1432
> * scale_factor: 1.5

I was doing the Scale too late in the resizing procedure. This should fix for whatever PPI the user device is.

File: lib/l10n/app_en.arb [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "@@locale": "en",
     "appTitle": "What to expect from Ubuntu on WSL",
     "windowTitle": "Installing Ubuntu on WSL",
-    "welcome": "Welcome to Ubuntu",
+    "welcome": "Welcome to Ubunt WSL",
```
Comment (jpnurmi) [inline lib/l10n/app_en.arb:5]: oops, missing "u" :smile:

Comment (CarlosNihelton) [reply lib/l10n/app_en.arb:5]: It's surpising how we become blind to those things... Thanks for spotting that out.

**Pull request merged**